src/sentry/processing/real_time_metrics/base.py

```diff
@@ -0,0 +1,42 @@
+import datetime
+from typing import Any
+
+from sentry.utils import redis
+
+
+class RealTimeMetricsStore:
+    def __init__(
+        self,
+        cluster: redis._RedisCluster,
+        counter_bucket_size: int,
+        counter_ttl: datetime.timedelta,
+        prefix: str = "symbolicate_event_low_priority",
+    ) -> None:
+        if counter_bucket_size <= 0:
+            raise ValueError("bucket size must be at least 1")
+
+        self.counter_bucket_size = counter_bucket_size
+        self.inner = cluster
+        self.counter_ttl: int = int(counter_ttl / datetime.timedelta(milliseconds=1))
+        self.prefix = prefix
+
+    def get(self, key: str) -> Any:
+        return self.inner.get(key)
```

Comment:
I'm not a massive fan of this.  Maybe make `self.inner` a public attribute so users can access the underlying cluster directly?  but really it seems this is used only by the tests?  so what was wrong in the tests with using the `redis_cluster` fixture directly?

Reply:
Ok, I'll revert it for now and revisit when I have an actual use case outside of tests ^^